Packages/com.unity.inputsystem/InputSystem/Devices/FakeScreenKeyboard.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections;
+
+namespace UnityEngine.InputSystem
+{
+    /// <summary>
+    /// Mock screen keyboard class, which simulates screen keyboard behavior and is validated by the tests in ScreenKeyboardTests.cs
+    /// </summary>
+    class FakeScreenKeyboard : ScreenKeyboard
```

Comment:
If I do this, there's no way for me to use screen keyboard in editor, sure it isn't visible in Editor, but you can use all the API and it will behave the same as runtime.   If I'll do the move, the user will be forced to write 

```
#if UNITY_EDITOR
...
#endif
```
code, which is not very nice.

Note: when https://docs.unity3d.com/ScriptReference/TouchScreenKeyboard.html is used in editor, it's faking the behavior

Reply:
(Gah... hitting every random button today... :D)

Good point. Makes sense to me. In that case, though, I wonder whether we should provide an *actual* emulation, though. I.e. a screen keyboard implemented in software. Otherwise, seems to not really meet the intention of keeping things going in the editor.

Reply:
Sure, I'll try to cook something quickly, make worth renaming this to EmulatedScreenKeyboard or EditorScreenKeyboard

Reply:
Here's my quick draft - https://www.youtube.com/watch?v=ISAv3A9NY2 

As far as I recall though, in one the meetings, someone suggested to make keyboard with UI elements (when it's ready), but for now I think this is better than nothing